code/modules/mob/living/simple_animal/abnormality/aleph/last_shot.dm

```diff
@@ -83,6 +83,14 @@ GLOBAL_LIST_EMPTY(meat_list)
 		spawn_cooldown = world.time + spawn_cooldown_time
 		MeatSpawn()
 
+	for(var/mob/living/carbon/human/H in view(3, src))
+		if(locate(/obj/structure/meatfloor) in get_turf(H))
+			return
+		new /obj/structure/meatfloor(get_turf(H))
+		H.adjustStaminaLoss(20, 0)
```

Comment:
This seems kind of lazy. Won't people just stand under the door to the cell now? Maybe you should just teleport the abnormality on a breach instead.

Reply:
Don't think they can, I can increase range another one.
I don't really like the teleport on breach; if only because that seems lazy...

Reply:
If you want to make it more interesting than other random teleport breaches, you can make it pick one of the nearest 4 or 5 xenospawns and make it spread a trail back to the cell or something.

Reply:
Is this just because you don't like Stam damage?

Reply:
Checking nearby xenospawns is very processor intensive; I'm going to teleport to department centre instead.